supertokens_python/recipe/passwordless/syncio/__init__.py

```diff
@@ -192,56 +194,74 @@ def list_codes_by_phone_number(
 
 
 def list_codes_by_device_id(
-    device_id: str, user_context: Union[None, Dict[str, Any]] = None
+    device_id: str,
+    tenant_id: Optional[str],
+    user_context: Union[None, Dict[str, Any]] = None,
 ) -> Union[DeviceType, None]:
     return sync(
-        asyncio.list_codes_by_device_id(device_id=device_id, user_context=user_context)
+        asyncio.list_codes_by_device_id(
+            device_id=device_id, tenant_id=tenant_id, user_context=user_context
+        )
     )
 
 
 def list_codes_by_pre_auth_session_id(
-    pre_auth_session_id: str, user_context: Union[None, Dict[str, Any]] = None
+    pre_auth_session_id: str,
+    tenant_id: Optional[str],
+    user_context: Union[None, Dict[str, Any]] = None,
 ) -> Union[DeviceType, None]:
     return sync(
         asyncio.list_codes_by_pre_auth_session_id(
-            pre_auth_session_id=pre_auth_session_id, user_context=user_context
+            pre_auth_session_id=pre_auth_session_id,
+            tenant_id=tenant_id,
+            user_context=user_context,
         )
     )
 
 
 def create_magic_link(
     email: Union[str, None],
     phone_number: Union[str, None],
+    tenant_id: Optional[str],
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> str:
     return sync(
         asyncio.create_magic_link(
-            email=email, phone_number=phone_number, user_context=user_context
+            email=email,
+            phone_number=phone_number,
+            tenant_id=tenant_id,
+            user_context=user_context,
         )
     )
 
 
 def signinup(
     email: Union[str, None],
     phone_number: Union[str, None],
+    tenant_id: Optional[str],
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> ConsumeCodeOkResult:
     return sync(
         asyncio.signinup(
-            email=email, phone_number=phone_number, user_context=user_context
+            email=email,
+            phone_number=phone_number,
+            tenant_id=tenant_id,
+            user_context=user_context,
         )
     )
 
 
 def send_email(
     input_: PasswordlessLoginEmailTemplateVars,
+    tenant_id: Optional[str],
```

Comment:
should be a part of PasswordlessLoginEmailTemplateVars

Reply:
Done